content/influxdb/v2.0/reference/glossary.md

```diff
@@ -259,9 +259,24 @@ Related entries: [bucket](#bucket)
 
 ### data type
 
-A data type is defined by the values it can take, the programming language used, or the operations that can be performed on it.
-
-InfluxDB supports the following data types: float, integer, string, boolean, and timestamp.
+A data type is defined by the values it can take, the programming language used, or the operations that can be performed on it. 
+
+InfluxDB supports the following data types: 
+- boolean 
+- dateTime
+- double 
+- field
+- float
+- ignored
+- integer
+- long 
+- measurement 
+- string
+- tag
+- timestamp 
+- unsignedLong 
```

Comment:
Some of these are just aliases for others. Fields, tags, measurements, and timestamps aren't "data types;" they're parts of an InfluxDB schema that, when using extended annotated CSV, define the datatype of the column (measurement = string, tag = string, timestamp = dateTime).

```suggestion
| Data type        | Alias/annotation   |
| :--------------- | :----------------- |
| string           |                    |
| boolean          |                    |
| float            | double             |
| integer          | int, long          |
| unsigned integer | uint, unsignedLong |
| time             | dateTime           |
```